index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+<meta charset=“UTF-8”>
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Oswald:wght@300&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="reset.css">
+<link rel="stylesheet" href="style.css">
+<title>Lista zakupów</title>
+</head>
+
+<body>
+<div class="top_box">
+	<div class="logo">
+	<p>Lista zakupów</p>
+	</div>
+</div>
+
+<div class="main_box">
```

Comment:
sematyka do ogarnieca, masz praktycznie same divy które Ci buduja strukture, w ogóle to nie jest semantyczne :)